europy/decorators.py

```diff
@@ -81,3 +94,42 @@ def minimum_functionality(name: str = "",
     labels: List[Union[str, TestLabel]] = [TestLabel.MINIMUM_FUNCTIONALITY]
 
     return decorator_factory(labels, name, description)
+
+
+
+
+def modeldetails(file_path: str = None):
```

Comment:
should this function name be `model_details` based on python function naming conventions?

Reply:
yeah good point, will update.